src/pages/ReportDetailsPage.tsx

```diff
@@ -371,6 +375,42 @@ function ReportDetailsPage({policies, report, route}: ReportDetailsPageProps) {
             });
         }
 
+        if (isTrackExpenseReport) {
```

Comment:
This check caused the following issue:
- https://github.com/Expensify/App/issues/52628

because it didn't account for `!isDeletedParentAction`, which was fixed in PR https://github.com/Expensify/App/pull/52786.